ast/ast.go

```diff
@@ -2051,6 +2053,29 @@ type DropTable struct {
 	Name     *Ident
 }
 
+// RenameTable is RENAME TABLE statement node.
+//
+//	RENAME TABLE {{.ToList | sqlJoin ", "}}
+type RenameTable struct {
+	// pos = Drop
+	// end = Name.end
+
+	Rename token.Pos // position of "RENAME" pseudo keyword
+
+	ToList []*RenameTableTo // len(ToList) > 0
+}
+
+// RenameTableTo is src TO dest node in RENAME TABLE statement.
+//
+//	{{.Src | sql}} TO {{.Dest | sql}}
+type RenameTableTo struct {
+	// pos = Src.pos
+	// end = Dest.end
+
+	Src  *Ident
+	Dest *Ident
```

Comment:
Why are you persistent with `src` and `dest` naming? The document says `old` and `new`, so we can follow it.

Reply:
Thank you. Fixed.